packages/contracts/contracts/L2/messaging/L2CrossDomainMessenger.sol

```diff
@@ -101,11 +101,10 @@ contract L2CrossDomainMessenger is
         bytes memory _message,
         uint256 _messageNonce
     )
-        nonReentrant
         public
     {
         require(
```

Comment:
I still do think that it would be better to alias the `l1CrossDomainMessenger` once up front which would make this comparison cheaper but not the biggest deal